web/app.js

```diff
@@ -2640,62 +2634,56 @@ function webViewerWheel(evt) {
         // OSs have different defaults for the number lines. But we generally
         // want one "clicky" roll of the wheel (which produces one event) to
         // adjust the zoom by one step.
-        if (Math.abs(delta) >= 1) {
-          ticks = Math.sign(delta);
-        } else {
-          // If we're getting fractional lines (I can't think of a scenario
-          // this might actually happen), be safe and use the accumulator.
-          ticks = PDFViewerApplication._accumulateTicks(
-            delta,
-            "_wheelUnusedTicks"
-          );
-        }
+        //
+        // If we're getting fractional lines (I can't think of a scenario
+        // this might actually happen), be safe and use the accumulator.
+        ticks =
+          Math.abs(delta) >= 1
+            ? Math.sign(delta)
+            : this._accumulateTicks(delta, "_wheelUnusedTicks");
```

Comment:
*Please note:* When replacing `PDFViewerApplication` with `this` here and letting Prettier auto-format the code, this code triggered the `unicorn/prefer-ternary` rule; hence these changes (which I think are overall nicer anyway).